### ISSUE TITLE
can you show, how to use this tool?

### ISSUE THREAD

**Issue by ricko2991** [hakistan/HakistanKeylogger#1]

**Comment by hakistan** [comment]
you can watch this video: https://www.youtube.com/watch?v=GXM8NCTmuZQ

**Comment by charlie-hackers** [comment]
Yeh key logger mene try kiyaa work nai kar rhaaa

**Comment by ricko2991** [comment]
Yeah, this is work perfectly, but how can i make the app manually through the source code ?

**Comment by doremi666** [comment]
@hakistan the Youtube link is not available?

**Comment by danialprogrammer** [comment]
hi . there is no apk to install . how can use this codes on android device???

**Comment by L4FeeR** [comment]
make video as public

**Comment by witoxd** [comment]
can you make the video public?

**Comment by JNylson** [comment]
> you can watch this video: https://www.youtube.com/watch?v=GXM8NCTmuZQ

This video is private
